css/index.css

```diff
@@ -0,0 +1,90 @@
+/* Font Import */
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@400;600&display=swap');
+
```

Comment:
Not A good Practice to include import in CSS file because code below the import code will execute after all the imports get load.

Reply:
Done.
Included google font's <link> in HTML file